packages/alfa-css/src/value/custom-ident.ts

```diff
@@ -0,0 +1,64 @@
+import { Parser } from "@siteimprove/alfa-parser";
+import { Predicate } from "@siteimprove/alfa-predicate";
+
+import { type Parser as CSSParser, Token } from "../syntax/index.js";
+
+import type { Resolvable } from "./resolvable.js";
+import { Ident } from "./ident.js";
+
+const { map, parseIf } = Parser;
+const { and } = Predicate;
+
+/**
+ * {@link https://developer.mozilla.org/en-US/docs/Web/CSS/custom-ident}
+ *
+ * @public
+ */
+export class CustomIdent
+  extends Ident<"custom-ident">
+  implements Resolvable<CustomIdent, never>
+{
+  public static of(value: string): CustomIdent {
+    return new CustomIdent(value);
+  }
+
+  private constructor(value: string) {
+    super("custom-ident", value);
+  }
+
+  public equals(value: unknown): value is this {
+    return value instanceof CustomIdent && super.equals(value);
+  }
+
+  public toJSON(): Ident.JSON<"custom-ident"> {
+    return {
+      ...super.toJSON(),
+    };
```

Comment:
```suggestion
    return super.toJSON();
```
No need to spread and rebuild the same object.
🤔 Which actually means this is maybe not even needed